tests/ignite/contrib/metrics/test_mean_absolute_relative_error.py

```diff
@@ -0,0 +1,65 @@
+from ignite.contrib.metrics import MeanAbsoluteRelativeError
+from ignite.exceptions import NotComputableError
+import torch
+from pytest import approx
+from pytest import raises
+
+
+def test_mean_absolute_relative_error():
+    a = torch.rand(4)
+    b = torch.rand(4)
+    c = torch.rand(4)
+    d = torch.rand(4)
+    ground_truth = torch.rand(4)
+
+    m = MeanAbsoluteRelativeError()
+    m.reset()
```

Comment:
I was discussed in some of previously proposed PRs that `reset` is not needed after the constructor. We can remove.